packages/statemanager/test/ethersStateManager.spec.ts

```diff
@@ -224,8 +224,8 @@ tape('runTx test: replay mainnet transactions', async (t) => {
 
     const blockTag = 15496077n
     common.setHardforkByBlockNumber(blockTag)
-    const txHash = '0xed1960aa7d0d7b567c946d94331dddb37a1c67f51f30bf51f256ea40db88cfb0'
-    const tx = await TransactionFactory.fromEthersProvider(provider, txHash, { common })
+    const txData = require('./testdata/providerData/transactions/0xed1960aa7d0d7b567c946d94331dddb37a1c67f51f30bf51f256ea40db88cfb0.json')
```

Comment:
These tests aren't focused on testing the `fetchFromProvider` calls in `util` and it was proving tricky to mock just the readonly `connection.url` property of the `ethers.JsonRpcProvider` interface.  We don't gain anything by testing the block/tx static constructors here as that coverage is elsewhere.  If you feel strongly, I can bring it back in but it will take some further messing about to get the `mockprovider` to work correctly on these tests if I do.